maquetado/setup/scss/titles.scss

```diff
@@ -0,0 +1,24 @@
+* {
```

Comment:
This should go in a different file, I think a good place would be `application.css`, that way it would cover the entire project